src/org/ods/util/CollectionWithForLoop.groovy

```diff
@@ -0,0 +1,30 @@
+package org.ods.util
+
+import com.cloudbees.groovy.cps.NonCPS
+
+class CollectionWithForLoop {
+
+    @NonCPS
```

Comment:
@s2oBCN please remove the NonCPS annotations from both methods. Closures might execute in a CPS context.